controllers/v1/cloud-services/files.js

```diff
@@ -0,0 +1,277 @@
+     * }]
+     */
+
+    /**
+      * Get Downloadable URL from cloud.
+      * @method
+      * @name getDownloadableUrl
+      * @param  {Request}  req  request body.
+      * @returns {JSON} Response with status and message.
+    */
+
+    async getDownloadableUrl(req) {
+        return new Promise(async (resolve, reject) => {
+
+            try {
+
+          
+                let downloadableUrl =
+                await filesHelpers.getDownloadableUrl(
+                     req.body.filePaths, 
+                     req.body.bucketName
+                );
+
+                return resolve({
+                    message: constants.apiResponses.CLOUD_SERVICE_SUCCESS_MESSAGE,
+                    result: downloadableUrl
+                })
+
+            } catch (error) {
+
+                return reject({
+                    status:
+                        error.status ||
+                        httpStatusCode["internal_server_error"].status,
+
+                    message:
+                        error.message
+                        || httpStatusCode["internal_server_error"].message,
+
+                    errorObject: error
+                })
+
+            }
+        })
+
+    }
+
+
+    /**
+     * @api {post} /kendra/api/v1/cloud-services/files/preSignedUrls  
+     * Get signed URL.
+     * @apiVersion 1.0.0
+     * @apiGroup Files
+     * @apiHeader {String} X-authenticated-user-token Authenticity token
+     * @apiParamExample {json} Request:
+     * {
+     * "fileNames" : [
+     * "N4X6E2/N4X6E2.png"
+     * ],
+     * "bucket":"sl-unnati-storage"
+     * }
+     * @apiSampleRequest /kendra/api/v1/cloud-services/files/preSignedUrls
+     * @apiUse successBody
+     * @apiUse errorBody
+     * @apiParamExample {json} Response:
+     * {
+     * "message": "Url generated successfully",
+     * "status": 200,
+     * "result": [
+     * {
+     * "file": "T9R6Y8/T9R6Y8.png",
+     * "url": "https://sl-unnati-storage.s3.ap-south-1.amazonaws.com/qrcode/T9R6Y8/T9R6Y8.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAJW4YWQMTNBKD2KTQ%2F20200421%2Fap-south-1%2Fs3%2Faws4_request&X-Amz-Date=20200421T024629Z&X-Amz-Expires=1800&X-Amz-Signature=81d593329c96b318f2924b876854e534bf80aef1f10ca80083d74188b46e69de&X-Amz-SignedHeaders=host",
+     * "payload": {
+     * "sourcePath": "T9R6Y8/T9R6Y8.png"
+     * },
+     * "cloudStorage": "AWS"
+     * }
+     * ]
+     * }
+     */
+
+    /**
+      * Get signed urls.
+      * @method
+      * @name preSignedUrls
+      * @param  {Request}  req  request body.
+      * @param  {Array}  req.body.fileNames - list of file names
+      * @param  {String}  req.body.bucket - name of the bucket  
+      * @returns {JSON} Response with status and message.
+    */
+
+   async preSignedUrls(req) {
```

Comment:
@aman-tunerlabs Please implement for different req body for submission and project